en/components/grids_templates/selection.md

```diff
@@ -363,34 +363,39 @@ Once done, we can make use of the [`igx-contrast-color`]({environment:sassApiUrl
 
 #### Create custom theme
  
-Next we create need to create a new theme that extends the [`igx-grid-theme`]({environment:sassApiUrl}/index.html#function-igx-grid-theme) passing our `text-color`, `background-color` and `border-yellow` variables as `$cell-selected-text-color`, `$cell-selected-background` and `$cell-active-border-color`, respectively:
+Next we create a new theme that extends the [`igx-grid-theme`]({environment:sassApiUrl}/index.html#function-igx-grid-theme) passing our `text-color`, `background-color` and `border-yellow` variables as `$cell-selected-text-color`, `$cell-selected-background` and `$cell-active-border-color`, respectively:
 
 ```scss
-$custom-grid-theme: igx-grid-paginator-theme(
+$custom-grid-theme: igx-grid-theme(
     $cell-selected-text-color: $text-color,
     $cell-active-border-color: $border-yellow,
     $cell-selected-background: $background-color
 );
 ```
 
-#### Consume theme
+#### Apply theme
 
-Afterwards, all we need to do is have our @@igSelector consume the newly created theme instead of the default one:
+Afterwards, all we need to do is include the mixin in our component's style (this could also be done in the `app.root`), so that our @@igSelector uses the newly created theme instead of the default one:
 
 ```scss
+    @include igx-grid($custom-grid-theme);
+```
+
+ >[!NOTE]
+ >If the component is using an [`Emulated ViewEncapsulation`](../themes/component-themes.md#view-encapsulation), it is necessary to penetrate this encapsulation using `::ng-deep`:
+
+ ```scss
     :host { 
         ::ng-deep {
             @include igx-grid($custom-grid-theme);
         }
     }
 ```
-
  >[!NOTE]
- >If the component is using an [`Emulated`](../themes/component-themes.md#view-encapsulation) ViewEncapsulation, it is necessary to `penetrate` this encapsulation using `::ng-deep`:
+ > With `Emulated` or `ShadowDОМ` view encapsulation, we scope the style under `:host` selector so as not to affect any other grids we might have in our application.
```

Comment:
Okay, not a fan of the two notes.
`:host` scoping is only really needed for `Emulated` and thus should probably be explained in the note above with it.
Basically, `::ng-deep` disables the emulated encapsulation (adding attrs) for its nested styles and Shadow DOM natively does the scoping already.